scripts/cv_api_master.py

```diff
@@ -0,0 +1,160 @@
+            'generate': ['keras_cv.layers.AutoContrast']
+        },
+        {
+            'path': 'channel_shuffle',
+            'title': 'ChannelShuffle layer',
+            'generate': ['keras_cv.layers.ChannelShuffle']
+        },
+        {
+            'path': 'cut_mix',
+            'title': 'CutMix layer',
+            'generate': ['keras_cv.layers.CutMix']
+        },
+        {
+            'path': 'equalization',
+            'title': 'Equalization layer',
+            'generate': ['keras_cv.layers.Equalization']
+        },
+        {
+            'path': 'fourier_mix',
+            'title': 'FourierMix layer',
+            'generate': ['keras_cv.layers.FourierMix']
+        },
+        {
+            'path': 'grayscale',
+            'title': 'Grayscale layer',
+            'generate': ['keras_cv.layers.Grayscale']
+        },
+        {
+            'path': 'grid_mask',
+            'title': 'GridMask layer',
+            'generate': ['keras_cv.layers.GridMask']
+        },
+        {
+            'path': 'mix_up',
+            'title': 'MixUp layer',
+            'generate': ['keras_cv.layers.MixUp']
+        },
+        {
+            'path': 'posterization',
+            'title': 'Posterization layer',
+            'generate': ['keras_cv.layers.Posterization']
+        },
+        {
+            'path': 'rand_augment',
+            'title': 'RandAugment layer',
+            'generate': ['keras_cv.layers.RandAugment']
+        },
+        {
+            'path': 'random_augmentation_pipeline',
+            'title': 'RandomAugmentationPipeline layer',
+            'generate': ['keras_cv.layers.RandomAugmentationPipeline']
+        },
+        {
+            'path': 'random_channel_shift',
+            'title': 'RandomChannelShift layer',
+            'generate': ['keras_cv.layers.RandomChannelShift']
+        },
+        {
+            'path': 'rancom_color_degeneration',
+            'title': 'RandomColorDegeneration layer',
+            'generate': ['keras_cv.layers.RandomColorDegeneration']
+        },
+        {
+            'path': 'random_cutout',
+            'title': 'RandomCutout layer',
+            'generate': ['keras_cv.layers.RandomCutout']
+        },
+        {
+            'path': 'random_hue',
+            'title': 'RandomHue layer',
+            'generate': ['keras_cv.layers.RandomHue']
+        },
+        {
+            'path': 'random_saturation',
+            'title': 'RandomSaturation layer',
+            'generate': ['keras_cv.layers.RandomSaturation']
+        },
+        {
+            'path': 'random_sharpness',
+            'title': 'RandomSharpness layer',
+            'generate': ['keras_cv.layers.RandomSharpness']
+        },
+        {
+            'path': 'random_shear',
+            'title': 'RandomShear layer',
+            'generate': ['keras_cv.layers.RandomShear']
+        },
+        {
+            'path': 'solarization',
+            'title': 'Solarization layer',
```

Comment:
Should it be `RandomSaturation` or only `Saturation`? All processing layers possess a `rate` param, which makes them random, doesn't it? I think KPL's name should be consistant.